src/gsi/fca_xtofca.f90

```diff
@@ -0,0 +1,128 @@
+module fca_xtofca_mod
```

Comment:
Clean up the comments, will review again.

Reply:
ok, comments cleaned up